nvflare/app_opt/flower/controller.py

```diff
@@ -97,3 +98,19 @@ def get_connector_config_params(self, fl_ctx: FLContext) -> dict:
         return {
             Constant.CONF_KEY_NUM_ROUNDS: self.num_rounds,
         }
+
+    def add_to_fed_job(self, job, ctx, flower_content: str):
+        """This method is required by Job API.
```

Comment:
regarding the docstring, this method is not required right? it is only needed if you want custom stuff for Job API "to".

Reply:
good point, will reword as it is not required